mne/evoked.py

```diff
@@ -116,6 +116,11 @@ def save(self, fname):
         ----------
         fname : string
             Name of the file where to save the data.
+            
+        Notes
+        -----
+        An evoked object contains a single condition only. To write
```

Comment:
What would be a good place to clarify that Evoked is single-condition, even though Epochs is not (and maybe add the dict comprehension for averaging multiple categories)? This confused me for awhile.
Should I put it into the Notes for the Epochs/Evoked classes?